extension/manifest.json

```diff
@@ -12,15 +12,18 @@
 			"24": "icons/wasp24.png",
 			"32": "icons/wasp32.png"
 		},
-		"default_title": "Apply gradients"
+		"default_title": "Apply gradients",
+		"default_popup": "menu/index.html"
```

Comment:
I started putting that in before you added the options page. It was so that the user could play around with settings _on_ the page they were planning to use the add-on on, and see it update in almost real-time.